tests/test_multiprocessing_atomic.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+'''
+tests.test_atomic
+'''
+
+from multiprocessing import Process
+
+import pytest
+
+import atomos.multiprocessing.atomic
+
+
+"""
+@pytest.fixture
```

Comment:
Okay so this means no dictionaries, for example? Seems like a pretty important limitation. I think we should update the README in this patch.